yogstation.dme

```diff
@@ -1466,6 +1466,7 @@
 #include "code\modules\admin\view_variables\topic_basic.dm"
 #include "code\modules\admin\view_variables\topic_list.dm"
 #include "code\modules\admin\view_variables\view_variables.dm"
+#include "code\modules\antagonists\sentient_creature.dm"
```

Comment:
?

Reply:
this I have no idea why that happen

Reply:
undo this change

Reply:
I dont know how...

Reply:
move it back to where it was

Reply:
I dont know where is the issue

Reply:
@ynot01 This happens when you open the project in DM, the include order got fucked at some point